# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/tps/tps/testrunner.py
# Commit: 2147c5c81edf
# Full Hash: 2147c5c81edf9cd56c19e7e92dfebfcd77329978
# Author: Ryan VanderMeulen <ryanvm@gmail.com>
# Date: 2023-03-01 17:52 -0500
# Regressor Bug: 1818349
# File Overlap Count: 24
# Overlapping Files: third_party/rust/sync15/src/client/coll_update.rs, third_party/rust/sync15/src/client/mod.rs, third_party/rust/sync15/.cargo-checksum.json, third_party/rust/sync15/src/client/sync.rs, Cargo.toml
# Description:
#   Backed out changesets b05343a5b533 and 57b2cda7e0db (bug 1818349) for causing bug 1819723.
# ==============================================================================

diff -r d59b76766f0d -r 2147c5c81edf testing/tps/tps/testrunner.py
--- a/testing/tps/tps/testrunner.py	Wed Mar 01 05:53:30 2023 +0000
+++ b/testing/tps/tps/testrunner.py	Wed Mar 01 17:52:01 2023 -0500
@@ -79,8 +79,6 @@
         # hrm - not sure what the release/beta channels will do?
         "xpinstall.signatures.required": False,
         "services.sync.testing.tps": True,
-        # removed data: schema restriction for easier testing of tabs
-        "services.sync.engine.tabs.filteredSchemes": "about|resource|chrome|file|blob|moz-extension",
         "engine.bookmarks.repair.enabled": False,
         "extensions.experiments.enabled": True,
         "webextensions.storage.sync.kinto": False,